recipes/font-ttf-source-code-pro-feedstock/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: font-ttf-source-code-pro
+  version: 2.030
+
+source:
+  url: https://github.com/adobe-fonts/source-code-pro/archive/2.030R-ro/1.050R-it.tar.gz
```

Comment:
This should be templated on the version number.

Reply:
Done